### ISSUE TITLE
Error when installing slides with zh_TW as default lang

### ISSUE THREAD

**Issue by joequant** [odoo/odoo#13704]
Impacted versions: 10.0  nightly build 20161006

Steps to reproduce: install slides

Current behavior:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 675, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 331, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/odoo/service/model.py", line 119, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 324, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 933, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python2.7/site-packages/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python2.7/site-packages/odoo/addons/web/controllers/main.py", line 866, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/lib/python2.7/site-packages/odoo/addons/web/controllers/main.py", line 854, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python2.7/site-packages/odoo/api.py", line 681, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python2.7/site-packages/odoo/api.py", line 672, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/odoo/addons/base/module/module.py", line 410, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/usr/lib/python2.7/site-packages/odoo/addons/base/module/module.py", line 484, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/usr/lib/python2.7/site-packages/odoo/modules/registry.py", line 78, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/usr/lib/python2.7/site-packages/odoo/modules/loading.py", line 337, in load_modules
    loaded_modules, update_module)
  File "/usr/lib/python2.7/site-packages/odoo/modules/loading.py", line 235, in load_marked_modules
    loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks)
  File "/usr/lib/python2.7/site-packages/odoo/modules/loading.py", line 159, in load_module_graph
    _load_data(cr, module_name, idref, mode, kind='demo')
  File "/usr/lib/python2.7/site-packages/odoo/modules/loading.py", line 95, in _load_data
    tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
  File "/usr/lib/python2.7/site-packages/odoo/tools/convert.py", line 851, in convert_file
    convert_xml_import(cr, module, fp, idref, mode, noupdate, report)
  File "/usr/lib/python2.7/site-packages/odoo/tools/convert.py", line 921, in convert_xml_import
    obj.parse(doc.getroot(), mode=mode)
  File "/usr/lib/python2.7/site-packages/odoo/tools/convert.py", line 802, in parse
    self.parse(rec, mode)
  File "/usr/lib/python2.7/site-packages/odoo/tools/convert.py", line 805, in parse
    self._tags[rec.tag](rec, de, mode=mode)
  File "/usr/lib/python2.7/site-packages/odoo/tools/convert.py", line 713, in _tag_record
    id = self.env(context=rec_context)['ir.model.data']._update(rec_model, self.module, res, rec_id or False, not self.isnoupdate(data_node), noupdate=self.isnoupdate(data_node), mode=self.mode)
  File "/usr/lib/python2.7/site-packages/odoo/addons/base/ir/ir_model.py", line 1159, in _update
    record = record.create(values)
  File "/usr/lib/python2.7/site-packages/odoo/addons/website_slides/models/slides.py", line 381, in create
    slide._post_publication()
  File "/usr/lib/python2.7/site-packages/odoo/addons/website_slides/models/slides.py", line 465, in _post_publication
    html_body = publish_template.with_context({'base_url': base_url}).render_template(publish_template.body_html, 'slide.slide', slide.id)
  File "/usr/lib/python2.7/site-packages/odoo/addons/mail/models/mail_template.py", line 363, in render_template
    raise UserError(_("Failed to render template %r using values %r")% (template, variables))
ParseError: "Failed to render template <Template memory:7f4b012c4d10> using values {'format_tz': <function <lambda> at 0x7f4b0126f1b8>, 'ctx': {'base_url': u'http://big-apple.local:8069'}, 'user': res.users(1,), 'object': slide.slide(6,)}
None" while parsing /usr/lib/python2.7/site-packages/odoo/addons/website_slides/data/website_slides_demo.xml:28, near
<record id="slide_0" model="slide.slide">
            <field name="name">Odoo Explained</field>
            <field name="url">http://www.youtube.com/watch?v=o0UM3Lg7K4Y</field>
            <field name="channel_id" ref="channel_public"/>
            <field name="category_id" ref="category_1"/>
            <field name="website_published" eval="True"/>
            <field name="tag_ids" eval="[(4, ref('tag2'))]"/>
        </record>
```
Expected behavior:
Should not have error

Video/Screenshot link (optional):

**Comment by joequant** [comment]
This is the stack track when I raise the raw exception.
The problem is that I'm using zh_TW locale, and so the format_tz creates a string which is outside
of the ascii range.

  File "/usr/lib/python2.7/site-packages/odoo/addons/mail/models/mail_template.py", line 353, in <lambda>
    'format_tz': lambda dt, tz=False, format=False, context=self._context: format_tz(self.env, dt, tz, format),
  File "/usr/lib/python2.7/site-packages/odoo/addons/mail/models/mail_template.py", line 49, in format_tz
    return "%s %s%s" % (fdate, ftime, (' (%s)' % tz) if tz else '')
ParseError: "ascii
04-24 \u4e0a\u5348
6
7
ordinal not in range(128)" while parsing /usr/lib/python2.7/site-packages/odoo/addons/website_slides/data/website_slides_demo.xml:28, near
<record id="slide_0" model="slide.slide">
            <field name="name">Odoo Explained</field>
            <field name="url">http://www.youtube.com/watch?v=o0UM3Lg7K4Y</field>
            <field name="channel_id" ref="channel_public"/>
            <field name="category_id" ref="category_1"/>
            <field name="website_published" eval="True"/>
            <field name="tag_ids" eval="[(4, ref('tag2'))]"/>
        </record>

**Comment by JKE-be** [comment]
Hello, thanks for the reporting

It has been fixed in v8 (by commit odoo/odoo@f4aae1c).
It will be available after the next forward port.

Best regards


JKE
